graphiti_core/utils/bulk_utils.py

```
@@ -140,12 +168,13 @@ async def add_nodes_and_edges_bulk_tx(
     for edge in entity_edges:
         if edge.fact_embedding is None:
             await edge.generate_embedding(embedder)
-
-    await tx.run(EPISODIC_NODE_SAVE_BULK, episodes=episodes)
+    await tx.run(EPISODIC_NODE_SAVE_BULK, episodes=episode)
+    ENTITY_NODE_SAVE_BULK = get_entity_node_save_bulk_query(nodes, driver.provider)
```

Ruff (0.11.9)

Redefinition of unused `ENTITY_NODE_SAVE_BULK` from line 39

(F811)